standard/result_or_error.lua

```diff
@@ -27,7 +37,10 @@ local parsedText = socketOrError
 		socketOrError:map(function(socket) socket:close() end)
 	end)
 	:get()
+```
 ]]
+---@class ResultOrError
+---@field is_a? function
```

Comment:
```suggestion
---@class ResultOrError: BaseClass
```

Reply:
the class module class uses as base for annos
https://github.com/Liquipedia/Lua-Modules/blob/34ee538b08dd1ba2223dbd4d33c09fb767430ddb/standard/class.lua#L19